Assets/Dev/Arthur/ArthurAssets/Highlight Mechanic/GenerateGrid.cs

```diff
@@ -0,0 +1,32 @@
+﻿//Attach this script to a GameObject to have it output messages when your mouse hovers over it.
```

Comment:
In the branch I pulled, there are two scripts that aren't coming through. Do you have something finished locally that isn't part of the PR? Here are the errors I'm getting

![2020-12-04_16h04_13](https://user-images.githubusercontent.com/33199129/101214660-5c5cd680-364a-11eb-84d4-25994e1cefcb.png)